conformance/tests/httproute-response-header-modifier.go

```diff
@@ -168,6 +168,43 @@ var HTTPRouteResponseHeaderModifier = suite.ConformanceTest{
 			},
 			Backend:   "infra-backend-v1",
 			Namespace: ns,
+		}, {
+			Request: http.Request{
+				Path: "/response-and-request-header-modifiers",
+				Headers: map[string]string{
+					"X-Header-Remove": "val",
+				},
+			},
+			BackendSetResponseHeaders: map[string]string{
+				"X-Header-Set-2":    "set-val-2",
+				"X-Header-Add-2":    "add-val-2",
+				"X-Header-Remove-2": "remove-val-2",
+				"Another-Header":    "another-header-val",
+				"X-Header-Remove-1": "val",
+			},
+			ExpectedRequest: &http.ExpectedRequest{
+				Request: http.Request{
+					Path: "/response-and-request-header-modifiers",
+					Headers: map[string]string{
+						"X-Header-Add": "add-appends-values",
+						"X-Header-Set": "set-overwrites-values",
+					},
+				},
+				AbsentHeaders: []string{"X-Header-Remove"},
+			},
+			Response: http.Response{
+				Headers: map[string]string{
+					"X-Header-Set-1": "header-set-1",
+					"X-Header-Set-2": "header-set-2",
+					"X-Header-Add-1": "header-add-1",
+					"X-Header-Add-2": "add-val-2,header-add-2",
+					"X-Header-Add-3": "header-add-3",
```

Comment:
seems like `X-Header-Add-3` is checking the same functionality as  `X-Header-Add-1` so we can remove one of them (both here and from the yaml)

Reply:
Done